### PR TITLE
Add how to use retrieveMaintenanceToken to readme

### PR DESCRIPTION
**Pull request by tommywong-okta**
#### Description:
Add section **Access token management** on how to use maintenance token
#### Testing details:
- [ ]  Verified basic functionality of change
- [ ]  Added tests 

#### Other considerations:
- [ ] Has security implications
- [ ] Has UX changes
- [ ] Version bump required
- [ ] Followed recommendations from the [Android Secure Coding Guidelines](https://oktawiki.atlassian.net/wiki/spaces/security/pages/2388757544/Android+Guidelines) 

##### RESOLVES: 
[OKTA-581364](https://oktainc.atlassian.net/browse/OKTA-581364)

### CODE REVIEW

**Review by FeiChen-okta — APPROVED** [review]

**Comment by mingxiajiang-okta** [comment]
@tommywong-okta we should update Readme for sample app as well: https://github.com/okta/okta-devices-kotlin/tree/master/push-sample-app , it still uses old scopes

**Review by FeiChen-okta — APPROVED** [review]

File: push-sample-app/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This example shows you how to use the [Okta Devices SDK](https://github.com/okta
 ## Prerequisites
 
 In order to use this project, your org's admin needs to:
-1. Add an OIDC app with the proper scopes (`okta.authenticators.manage.self`, `okta.authenticators.read` and `okta.users.read.self`).
+1. Add an OIDC app with the proper scopes (`okta.myAccount.appAuthenticator.manage`, `okta.myAccount.appAuthenticator.read` and `openid`).
```
Comment (mingxiajiang-okta) [inline push-sample-app/README.md:17]: @tommywong-okta what's openid scope in OIDC app? I cannot find it from OKTA API scopes
image

Comment (tommywong-okta) [reply push-sample-app/README.md:17]: It is standard oidc scope not okta api scope

Comment (mingxiajiang-okta) [reply push-sample-app/README.md:17]: So org admin will know what to config to enable the scope openid when comes to "1. Add an OIDC app with the proper scopes (`okta.myAccount.appAuthenticator.manage`, `okta.myAccount.appAuthenticator.read` and `openid`).", right?

Comment (tommywong-okta) [reply push-sample-app/README.md:17]: @IldarAbdullin-okta Since this message is for org admin, should we only include okta api scope here?

Comment (IldarAbdullin-okta) [reply push-sample-app/README.md:17]: It is for the dev, not for org admin. Openid is a standard oidc scope which is always enabled for all authz servers

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM!

File: push-sample-app/README.md (hunk 1 above)
Comment (mingxiajiang-okta) [reply push-sample-app/README.md:17]: @tommywong-okta @IldarAbdullin-okta should we remove openid from here if it is not for org admin?

**Comment by FeiChen-okta** [comment]
@tommywong-okta you can merge this since this is same as iOS and brent approved the iOS readme.

**Pull request merged**